OpenRA.Test/Fakes.cs

```diff
@@ -0,0 +1,206 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OpenRA.Mods.Common.Traits;
+
+namespace OpenRA.Test
+{
+	public class FakeActor : IActor
+	{
+		IWorld world;
+
+		public ActorInfo Info
+		{
+			get { throw new NotImplementedException("No need to implement this yet"); }
```

Comment:
The problem is the instantiation of ActorInfo on a unit test without interfacing it.

Anyway, In the next PR about Move I'll get rid of the fakes classes in favor of the Moq framework or at least refactor them heavily, so I don't feel like wasting time here for now.